bin/main.rb

```diff
@@ -1 +1,72 @@
-puts 'Hello World!'
\ No newline at end of file
+#!/usr/bin/ruby
+
+class Player
+  attr_accessor :name, :taken
+  def initialize(name = 'Player')
+    @name = name
+    @taken = []
+  end
+end
+
+class DisplayInterface
+  attr_accessor :board
+  def initialize
+    @board = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+  end
+
+  def take_place(inpt, play)
+    @board[inpt - 1] = play if play == 'O' || play == 'X'
+  end
```

Comment:
- Board can be approached as a component (an object) which is independent from the DisplayInterface and has it's own behaviour (methods and attributes)

Reply:
We have implemented the changes @stratospherique